frontend/src/eventPage/Expenditures.js

```diff
@@ -11,8 +11,8 @@ import {Button} from "@material-ui/core";
 
 export default function Expenditures() {
   const { events } = useContext(EventContext);
-  const { id } = useParams();
-  const event = events.find((event) => event.id === id);
+  const { eventId } = useParams();
+  const event = events.find((event) => event.id === eventId);
```

Comment:
This pattern is reapeated multiple times, maybe write a custom hook to reduce duplication?

```
const useEvent = () => {
    const { events } = useContext(EventContext);
    const { eventId } = useParams();
    const event = events.find((event) => event.id === eventId);
    return event;
}
```